### DISCUSSION TITLE
Error: SQLITE_CANTOPEN: unable to open database file

### DISCUSSION THREAD

**Discussion by StefanZivkovic** [vercel/community#161] [Help] [answered]
I picked **Vercel** and I am trying to host node service which does slack messages on schedule (cron job).
Does anyone by chance know what could be the reason for issues in case of using `sqlite `and `typeorm `for setting up DB?
Currently i am having issue `Error: SQLITE_CANTOPEN: unable to open database file`, and i have tried with `path.resolve(__dirname, 'sqlite.db')` or just `sqlite.db` but it does not work

**Comment by leerob** [comment, marked as answer]
If you're trying to read from your SQLite DB file, you'll need to explicitly include it in your function: https://vercel.com/support/articles/how-can-i-use-files-in-serverless-functions

However, you won't be able to do full CRUD options (just read from the file). Instead, I would recommend checking out a managed MySQL provider: https://vercel.com/integrations/planetscale

**Reply by StefanZivkovic** [reply]
Thanks for reply. I will follow your links and try it out in next few days. And announce the output (for future readers).
